henryPuma/assignments/twopointers/w1d3/lc26.js

```diff
@@ -0,0 +1,53 @@
+// Problem Statement #
```

Comment:
Nice work on the solution but again would be great to see a detailed pseudocode overview and/or some more detailed comments about your approach.